### ISSUE TITLE
Using not supported OAuth 1.0 signature

### ISSUE THREAD

**Issue by postatum** [aml-org/amf#491]
### Issue
Running code below should fail or report validation error because according to RAML 1.0 spec provided RAML input is invalid.

The spec says:
> A list of signature methods used by the Authorization server, which can be any of the following: HMAC-SHA1, RSA-SHA1, or PLAINTEXT

Spec link:
https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/#oauth-10

### AMF Version
4.0.0-SNAPSHOT

### Code
```sh
$ java -jar amf-4.0.0-SNAPSHOT.jar validate -in "RAML 1.0" -mime-in "application/yaml" -p "RAML" file://api.raml
{
  "@type": "http://www.w3.org/ns/shacl#ValidationReport",
  "http://www.w3.org/ns/shacl#conforms": true
}
```

### api.raml
```raml
#%RAML 1.0
title: API
version: 1
baseUri: https://api.dropbox.com/{version}
securitySchemes:
  oauth_1_0:
    description: |
      supports OAuth 1.0 for authenticating all API requests.
    type: OAuth 1.0
    settings:
      requestTokenUri: https://api.mysampleapi.com/1/oauth/request_token
      tokenCredentialsUri: https://api.mysampleapi.com/1/oauth/access_token
      authorizationUri: https://api.mysampleapi.com/1/oauth/authorize
      signatures: [ 'HI' ]
```

**Comment by gutee** [comment]
Jira issue to track: https://www.mulesoft.org/jira/browse/APIMF-1522

**Comment by gutee** [comment]
This is fixed and stated for 3.3.0 release
